### PR TITLE
Bump core dependency

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: sdk/core/Azure.Core.Experimental/CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,11 @@
 # Release History
 
-## 0.1.0-preview.26 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 0.1.0-preview.26 (2023-04-10)
```
Comment (annelo-msft) [inline sdk/core/Azure.Core.Experimental/CHANGELOG.md:3]:
```suggestion
## 0.1.0-preview.26 (2023-04-10)

### Features Added

- Added basic debugger support for DynamicData
```

File: sdk/core/Azure.Core.Experimental/CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
 
 ### Bugs Fixed
 
 - Use specified `DynamicJsonNameMapping` in array and object enumerators.
```
Comment (annelo-msft) [inline sdk/core/Azure.Core.Experimental/CHANGELOG.md:11]:
```suggestion
- Use `DynamicJsonOptions` in child elements, arrays, and object enumerators.
```

File: sdk/core/Azure.Core.Experimental/CHANGELOG.md [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
 
-### Breaking Changes
+- Added basic debugger support for DynamicData
 
 ### Bugs Fixed
 
-- Use specified `DynamicJsonNameMapping` in array and object enumerators.
-
-### Other Changes
+- Use `DynamicJsonOptions` in child elements, arrays, and object enumerators.
```
Comment (annelo-msft) [inline sdk/core/Azure.Core.Experimental/CHANGELOG.md:11]:
```suggestion
- Use root `DynamicJsonOptions` in child elements, arrays, and object enumerators.
```

Comment (JoshLove-msft) [reply sdk/core/Azure.Core.Experimental/CHANGELOG.md:11]: I think you added this in your other comment :)

File: sdk/core/Azure.Core.Experimental/CHANGELOG.md [hunk 4]
```diff
@@ -1,16 +1,14 @@
 # Release History
 
-## 0.1.0-preview.26 (Unreleased)
+## 0.1.0-preview.26 (2023-04-10)
 
 ### Features Added
 
-### Breaking Changes
+- Added basic debugger support for DynamicData
```
Comment (annelo-msft) [inline sdk/core/Azure.Core.Experimental/CHANGELOG.md:8]:
```suggestion
### Breaking Changes

- Removed `DynamicJson` type, and moved its functionality into `DynamicData`.
- Renamed `DynamicJsonNameMapping` enum to `DynamicDataNameMapping`.
- Renamed `DynamicJsonProperty` to `DynamicDataProperty`.
- Renamed `ToDynamic()` extension method on `BinaryData` to `ToDynamicFromJson()`.
- Removed `DynamicJson.ArrayEnumerator` and `DynamicJson.ObjectEnumerator` types.
```

File: sdk/core/Azure.Core.Experimental/CHANGELOG.md [hunk 5]
```diff
@@ -1,16 +1,20 @@
...
 
-## 0.1.0-preview.26 (Unreleased)
+## 0.1.0-preview.26 (2023-04-10)
 
 ### Features Added
 
+- Added basic debugger support for DynamicData
 ### Breaking Changes
 
+- Removed `DynamicJson` type, and moved its functionality into `DynamicData`.
```
Comment (annelo-msft) [inline sdk/core/Azure.Core.Experimental/CHANGELOG.md:10]:
```suggestion
- Removed `DynamicJson` type, and moved its functionality into `DynamicData`.
- Sealed the `DynamicData` type.
```

**Review by annelo-msft — APPROVED** [review]

**Review by christothes — APPROVED** [review]

**Pull request merged**